File: bls_jztc/demo/internal/model/payment.go
package model

// PaymentConfig 支付配置模型
type PaymentConfig struct {
	// 微信支付配置
	AppId     string `json:"appId"`     // 微信支付AppID
	MchId     string `json:"mchId"`     // 微信支付商户号
	ApiKey    string `json:"apiKey"`    // API密钥
	NotifyUrl string `json:"notifyUrl"` // 回调通知地址
	IsEnabled bool   `json:"isEnabled"` // 是否启用微信支付
}

File: bls_jztc/jztc/src/api/payment.ts
import request from '../utils/request';

// 支付配置接口响应
export interface PaymentConfigResponse {
  code?: number;
  message?: string;
  data?: {
    appId: string;
    mchId: string;
    apiKey: string;
    notifyUrl: string;
    isEnabled: boolean;
  };
  appId?: string;
  mchId?: string;
  apiKey?: string;
  notifyUrl?: string;
  isEnabled?: boolean;
}

// 保存配置请求参数
export interface SavePaymentConfigParams {
  appId: string;
  mchId: string;
  apiKey: string;
  notifyUrl: string;
  isEnabled?: boolean;
}

// 保存配置响应
export interface SavePaymentConfigResponse {
  code?: number;
  message?: string;
  data?: {
    success: boolean;
    responseMessage: string;
  };
  success?: boolean;
  responseMessage?: string;
}

/**
 * 获取支付配置
 * @returns 支付配置信息
 */
export const getPaymentConfig = (): Promise<PaymentConfigResponse> => {
  return request.get('/payment/config');
};

/**
 * 保存支付配置
 * @param params 支付配置参数
 * @returns 保存结果
 */
export const savePaymentConfig = (params: SavePaymentConfigParams): Promise<SavePaymentConfigResponse> => {
  return request.post('/payment/config', params);
}; 